src/app/pages/repositoriesSearch/controllers/_repositoriesSearch.js

```diff
@@ -0,0 +1,17 @@
+/* @ngInject */
+const repositoriesSearch = ($state, $scope, KeyCodes) => {
+    $scope.getRepositoriesInfo = getRepositoriesInfo;
+    $scope.keyCodes = KeyCodes;
+
+    init();
+
+    function init() {}
+
+    function getRepositoriesInfo() {
+        $state.go('repositoriesSearch.result', {
```

Comment:
What about pressing enter key?

Reply:
Please check this link
https://stackoverflow.com/questions/22124663/trigger-state-in-button-tag-using-ui-sref
I'm ok to leave it as is, but please see which solution fits the case better

Reply:
I already know about that. But this solution does not suit for me because i have to change state not only on button click but also when i press enter key inside search input control.